File: src/jane/handler/testclient/TestEmptyHandler.java
package jane.handler.testclient;

import org.apache.mina.core.session.IoSession;
import jane.bean.TestEmpty;
import jane.core.BeanHandler;
import jane.core.Log;
import jane.core.NetManager;

public final class TestEmptyHandler extends BeanHandler<TestEmpty>
{
	/*\
	\*/

	@Override
	public void onProcess(NetManager manager, IoSession session, TestEmpty arg)
	{
		Log.log.debug("{}.onProcess: arg={}", getClass().getName(), arg);
	}
}

File: tool/csharp/jane/handler/testclient/TestEmptyHandler.cs
using jane.bean;

namespace jane.handler.testclient
{
	public class TestEmptyHandler : BeanHandler<TestEmpty>
	{
		/*\
		\*/

		public override void onProcess(object manager, object session, TestEmpty arg)
		{
			// Log.log.debug("{}.onProcess: arg={}", getClass().getName(), arg);
		}
	}
}
